components/eamxx/cime_config/testdefs/testmods_dirs/scream/kokkos_rrtmgp/shell_commands

```diff
@@ -0,0 +1,3 @@
+./xmlchange --append SCREAM_CMAKE_OPTIONS='SCREAM_RRTMGP_ENABLE_YAKL Off'
```

Comment:
Since Kokkos is the default, do we need this testmod?

Reply:
Probably not unless you want to be double-sure that Kokkos is on :)